sdk/confidentialledger/azure-confidentialledger/azure/confidentialledger/_client.py

```diff
@@ -0,0 +1,480 @@
+            ledger_base_url=self._ledger_url, **kwargs
+        )
+        return Consortium(
+            members=[
+                ConsortiumMember(certificate=member.certificate, member_id=member.id)
+                for member in result.members
+            ]
+        )
+
+    @distributed_trace
+    def get_enclave_quotes(
+        self,
+        **kwargs,  # type: Any
+    ):
+        # type: (...) -> LedgerEnclaves
+        """Gets enclave quotes from all nodes in the Confidential Ledger network.
+
+        :return: Enclave quotes for nodes in the Confidential Ledger.
+        :rtype: ~azure.confidentialledger.LedgerEnclaves
+        :raises: ~azure.core.exceptions.HttpResponseError
+        """
+
+        result = self._client.get_enclave_quotes(
+            ledger_base_url=self._ledger_url, **kwargs
+        )
+        return LedgerEnclaves(
+            {
+                quote.node_id: EnclaveQuote(
+                    node_id=quote.node_id,
+                    mrenclave=quote.mrenclave,
+                    raw_quote=quote.raw,
+                    version=quote.quote_version,
+                )
+                for _, quote in result.enclave_quotes.items()
+            },
+            result.current_node_id,
+        )
+
+    @distributed_trace
+    def get_ledger_entries(
+        self,
+        **kwargs,  # type: Any
+    ):
+        # type: (...) -> ItemPaged[LedgerEntry]
+        """Gets a range of entries in the ledger.
+
+        :keyword str from_transaction_id: Transaction identifier from which to start the query.
+            If this is not specified, the query begins from the first transaction.
+        :keyword str to_transaction_id: Transaction identifier at which to end the query
+            (inclusive). If this is not specified, the query ends at the end of the ledger.
+        :keyword str sub_ledger_id: Identifies the sub-ledger to fetch the ledger entry from.
+        :return: An iterable for iterating over the entries in the range.
+        :rtype: ~azure.core.paging.ItemPaged[LedgerEntry]
+        :raises: ~azure.core.exceptions.HttpResponseError
+        """
+
+        from_transaction_id = kwargs.pop("from_transaction_id", None)
+        to_transaction_id = kwargs.pop("to_transaction_id", None)
+        sub_ledger_id = kwargs.pop("sub_ledger_id", None)
+
+        return self._client.get_ledger_entries(
+            ledger_base_url=self._ledger_url,
+            from_transaction_id=from_transaction_id,
+            to_transaction_id=to_transaction_id,
+            sub_ledger_id=sub_ledger_id,
+            cls=kwargs.pop(
+                "cls",
+                lambda entries: [
+                    LedgerEntry._from_pipeline_result(entry) for entry in entries
+                ]
+                if entries is not None
+                else [],
+            ),
+            **kwargs,
+        )
+
+    @distributed_trace
+    def get_ledger_entry(
+        self,
+        interval=0.5,  # type: float
```

Comment:
Not sure about the desired API design, but I think you can make `interval` and `max_tries` kwargs which default to the values you have here. cc @johanste though, def deferring to him

Reply:
this applies to all of the `interval` and `max_tries` params

Reply:
I didn't include them in `kwargs` so it's a bit more explicitly configurable to the user, but I'm open to the change if it's prescribed.

Reply:
Sure, I'll make it keyword only then. I don't think it makes sense to be configured on the client itself, because these timeouts/retries only apply to a select few methods, so it may be confusing for the user to understand when they are used.